sass/_CardActionBar.scss

```diff
@@ -0,0 +1,41 @@
+.CoveoCardActionBar {
```

Comment:
Looking at https://coveo-search-ui-pr-230.herokuapp.com/#t=All&sort=relevancy&layout=card. Very nice! 
Is there or could there be an option to open the `CoveoCardActionBar` on hover rather than on a click? 
With a click, I find that the QuickView is not that easily discoverable.